lib/Tomcat/JspTest.pm

```diff
@@ -17,30 +17,50 @@ use warnings;
 use testapi;
 use utils;
 use Tomcat::Utils;
+use version_utils 'is_sle';
 
 # allow a 60 second timeout for asserting needles
 use constant TIMEOUT => 60;
 
 # test all JSP examples
 sub test_all_examples() {
-    my ($self) = shift;
+    my ($self, $mod_jk) = @_;
 
     # array with example test function and number of tabs required to select the example
     # shuffle example is skipped
-    my @jsp_examples = ([\&basic_arithmetics, 2], [\&basic_comparisons, 4], [\&implicit_objects, 4], [\&functions, 4], [\&composite_expressions, 4], [\&hello_world_tag, 4], [\&repeat_simple_tag, 4], [\&book_tag, 4], [\&tag_file, 4], [\&panel_tag, 4], [\&display_product, 4], [\&xhtml_basic, 4], [\&attribute_body, 8], [\&dynamic_attributes, 8], [\&jsp_configuration, 4], [\&number_guess, 4], [\&date, 4], [\&snoop, 4], [\&error, 4], [\&carts, 4], [\&checkbox, 4], [\&color, 4], [\&calendar, 4], [\&include, 4], [\&forward, 4], [\&plugin, 4], [\&servlet_jsp, 4], [\&simple_tag, 4], [\&jsp_xml, 4], [\&if, 4], [\&foreach, 4], [\&choose, 4], [\&form, 4]);
+    my @jsp_examples = ([\&basic_arithmetics, 2], [\&basic_comparisons, 4], [\&implicit_objects, 4], [\&functions, 4], [\&composite_expressions, 4], [\&hello_world_tag, 4]);
```

Comment:
Are those other functions (repeat_simple_tag, book_tag, tag_file, panel_tag, display_product etc) failing with $mod_jk? Or is it that that that tomcat functionality is not there in that case?

Reply:
As per my understanding its the functionality, Static files in the examples webapp are served by Apache. I had refered the test steps from https://qam.suse.de/testreports/SUSE:Maintenance:15126:218530/log.